lib/cli/commands/getinfo.ts

```diff
@@ -3,109 +3,87 @@ import { Arguments } from 'yargs';
 import Table, { VerticalTable } from 'cli-table3';
 import colors from 'colors/safe';
 import { GetInfoRequest, GetInfoResponse, LndInfo, RaidenInfo } from '../../proto/xudrpc_pb';
+import { SwapClientStatus } from '../../constants/enums';
 
 type generalInfo = {
+  alias: string;
+  address: string;
+  network: string;
   version: string;
   numPeers: number;
   numPairs: number;
   nodePubKey: string;
   orders: {own: number, peer: number} | undefined
 };
 
-const displayChannels = (channels: any, asset: string) => {
-  const table = new Table() as VerticalTable;
-  Object.keys(channels).forEach((key: any) => {
-    table.push({
-      [colors.blue(key)] : channels[key],
-    });
-  });
-  console.log(colors.underline(colors.bold(`\nLnd ${asset} channels:`)));
-  console.log(table.toString(), '\n');
-};
-
-const displayChainsList = (list: any[], asset: string) => {
-  const table = new Table() as VerticalTable;
-  list.forEach((asset, i) => {
-    if (asset) {
-      table.push({ [colors.blue(`${i + 1}.`)]: `${asset.chain}-${asset.network}` });
-    }
-  });
-  if (table.length !== 0) {
-    console.log(colors.underline(colors.bold(`\nLnd ${asset} chains:`)));
-    console.log(table.toString(), '\n');
-  }
-};
-
-const displayUriList = (uris: string[], asset: string) => {
-  const table = new Table() as VerticalTable;
-  uris.forEach((uri, i) => table.push({ [`${i + 1}.`]: uri }));
-  console.log(colors.underline(colors.bold(`\nLnd ${asset} uris:`)));
-  console.log(table.toString(), '\n');
-};
-
 const displayLndInfo = (asset: string, info: LndInfo.AsObject) => {
   const basicInfotable = new Table() as VerticalTable;
   basicInfotable.push(
-    { [colors.blue('Error')]: info.error },
+    { [colors.blue('Status')]: SwapClientStatus[info.status] },
   );
-  if (info.blockheight) {
-    basicInfotable.push({ [colors.blue('Block Height')]: info.blockheight });
-  }
-  if (info.version) {
-    basicInfotable.push({ [colors.blue('Version')]: info.version });
-  }
-  if (info.alias) {
-    basicInfotable.push({ [colors.blue('Alias')] : info.alias });
+  if (info.error) {
+    basicInfotable.push(
+        { [colors.blue('Error')]: info.error },
+    );
   }
 
-  console.log(colors.underline(colors.bold(`\nLnd ${asset} info:`)));
-  console.log(basicInfotable.toString(), '\n');
-
-  if (info.channels) {
-    displayChannels(info.channels, asset);
-  }
+  const address = info.urisList[0] ? `${info.urisList[0].substring(0, info.urisList[0].indexOf('@'))}
+${info.urisList[0].substring(info.urisList[0].indexOf('@'))}` : '';
 
-  if (!info.error) {
-    displayChainsList(info.chainsList, asset);
-  }
+  basicInfotable.push(
+    { [colors.blue('Version')]: info.version ? info.version : ''   },
```

Comment:
```suggestion
    { [colors.blue('Version')]: info.version || ''   },
```
This can be simplified a little